Makefile

```diff
@@ -0,0 +1,27 @@
+PREFIX ?= ~/.local/share/gnome-shell/extensions
+
+TRANSLATION_ID=clipboard-indicator
+EXTENSION_UUID=clipboard-indicator@tudmotu.com
+
+all: build install
```

Comment:
I'd drop the `install`, assuming most people do not expect a `install` when just running `make`. Also, maybe move this to a own commit?